examples/multi-token/mt/src/lib.rs

```diff
@@ -0,0 +1,453 @@
+use near_contract_standards::multi_token::metadata::MT_METADATA_SPEC;
+use near_contract_standards::multi_token::token::{ClearedApproval, Token, TokenId};
+use near_contract_standards::multi_token::{
+    core::MultiToken,
+    metadata::{MtContractMetadata, TokenMetadata},
+};
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::collections::LazyOption;
+use near_sdk::json_types::U128;
+use near_sdk::Promise;
+use near_sdk::{
+    env, near_bindgen, require, AccountId, BorshStorageKey, PanicOnDefault, PromiseOrValue,
+};
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize, PanicOnDefault)]
+pub struct ExampleMTContract {
+    tokens: MultiToken,
+    metadata: LazyOption<MtContractMetadata>,
+}
+
+#[derive(BorshSerialize, BorshStorageKey)]
+enum StorageKey {
+    MultiToken,
+    Metadata,
+    TokenMetadata,
+    Enumeration,
+    Approval,
+}
+
+#[near_bindgen]
+impl ExampleMTContract {
+    #[init]
+    pub fn new_default_meta(owner_id: AccountId) -> Self {
+        let metadata = MtContractMetadata {
+            spec: MT_METADATA_SPEC.to_string(),
+            name: "Example NEAR multi token".to_string(),
+            symbol: "EXAMPLE".to_string(),
+            icon: None,
+            base_uri: None,
+            reference: None,
+            reference_hash: None,
+        };
+
+        Self::new(owner_id, metadata)
+    }
+
+    #[init]
+    pub fn new(owner_id: AccountId, metadata: MtContractMetadata) -> Self {
+        require!(!env::state_exists(), "Already initialized");
```

Comment:
Since [near-sdk-rs v3.0.1](https://github.com/near/near-sdk-rs/blob/b8fc60809b907543d909ee75fcc1bd7b68cbe2fd/CHANGELOG.md#301---2021-03-25):

> BREAKING `#[init]` now checks that the state is not initialized. This is expected behavior. To ignore state check you can call `#[init(ignore_state)]`

Thus, you don't need this check:

```suggestion
```

Reply:
removed this